### PR TITLE
add mlavacca as ingress2gateway maintainer

### PR DESCRIPTION
**Pull request by LiorLieberman**
As discussed offline, we are adding Mattia as ingress2gateway maintainer

/cc @shaneutt @robscott @youngnick @mlavacca

### CODE REVIEW

**Comment by shaneutt** [comment]
Looks good. While you're in here, please feel free to move me to `emeritus`.

**Comment by robscott** [comment]
LGTM, thanks @mlavacca and @LiorLieberman!

We should probably also move Bowei to emeritus.

**Comment by LiorLieberman** [comment]
Removed from here and added `emeriturs_approvers` on ingress2gateway OWNERS - https://github.com/kubernetes-sigs/ingress2gateway/pull/149

File: config/kubernetes-sigs/sig-network/teams.yaml [hunk 1]
```diff
@@ -114,10 +115,9 @@ teams:
   ingress2gateway-maintainers:
     description: Write access to the ingress2gateway repo
     members:
-    - bowei
     - LiorLieberman
+    - mlavacca
     - robscott
-    - shaneutt
```
Comment (shaneutt) [inline config/kubernetes-sigs/sig-network/teams.yaml:120]: You've _removed_ previous maintainers, what we do instead is we'll add a section called `emeritus` which doesn't carry any official position but denotes previously involved maintainers.

**Review by shaneutt — CHANGES_REQUESTED** [review]

**Review by shaneutt — APPROVED** [review]
/approve

**Pull request merged**